packages/grafana-data/src/utils/nodeGraph.ts

```diff
@@ -38,4 +38,10 @@ export enum NodeGraphDataFrameFieldNames {
 
   // Defines the stroke dash array for the edge [edges]. See SVG strokeDasharray definition for syntax.
   strokeDasharray = 'strokedasharray',
+
+  // Supplies a fixed X position for the node to have in the finished graph.
+  fixedX = 'fixedX',
+
+  // Supplies a fixed Y position for the node to have in the finished graph.
+  fixedY = 'fixedY',
```

Comment:
I am not exactly sure why, I think we have some problems with the whole case insensitivness checks but all other values are lowercase so lets also do the same here `fixedY = 'fixedy'`